server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -762,6 +764,17 @@ public void execute(final String[] args) {
       if (opts.resetSecurity) {
         AccumuloServerContext context = new AccumuloServerContext(new ServerConfigurationFactory(HdfsZooInstance.getInstance()));
         if (isInitialized(fs)) {
+          if (!opts.forceResetSecurity) {
+            String secret = acuConf.get(Property.INSTANCE_SECRET);
+            ConsoleReader c = getConsoleReader();
+            String userEnteredSecret = c.readLine("WARNING: This will remove all users from Accumulo! If you wish to proceed enter the instance secret: ", '*');
```

Comment:
I am open to suggestions for the wording of the message, I was going by what was requested in the ticket.  I believe having the user enter something other than Y/N was to require additional thought as to whether they really want perform this action.